src/components/Project/ProjectHeader/SocialLinks.tsx

```diff
@@ -1,51 +1,87 @@
-import { TwitterOutlined } from '@ant-design/icons'
+import { TwitterOutlined, GlobalOutlined } from '@ant-design/icons'
 import { Space } from 'antd'
 import ExternalLink from 'components/ExternalLink'
 import Discord from 'components/icons/Discord'
-import { linkUrl, prettyUrl } from 'utils/url'
+import Telegram from 'components/icons/Telegram'
+import { useJuiceTheme } from 'hooks/JuiceTheme'
+import useMobile from 'hooks/Mobile'
+import colors from 'tailwindcss/colors'
+
+type SocialProps = {
+  children: React.ReactNode
+  link: string
+}
+
+function SocialButton(props: SocialProps) {
```

Comment:
these buttons are just used in this one file and thought co-locating the button component was easier to grok.